react/features/video-menu/actions.any.ts

```diff
@@ -11,7 +11,7 @@ import { shouldShowModeratedNotification } from '../av-moderation/functions';
 import { setAudioMuted, setVideoMuted } from '../base/media/actions';
 import { MEDIA_TYPE, MediaType, VIDEO_MUTISM_AUTHORITY } from '../base/media/constants';
 import { muteRemoteParticipant } from '../base/participants/actions';
-import { getLocalParticipant, getRemoteParticipants } from '../base/participants/functions';
+import { getRemoteParticipants } from '../base/participants/functions';
```

Comment:
I don't see why do we need this change?

Reply:
Ups, that needs to be in the first commit. "[fix(video-menu): When muting all skip local.](https://github.com/jitsi/jitsi-meet/pull/15628/commits/1928e64515b3f6f28df30afaae8609b804b7f9eb)"
I can move it.

Reply:
it was failing lint and I put it in the wrong commit.

Reply:
Done.